state/azure/cosmosdb/storedprocedure.go

```diff
@@ -0,0 +1,118 @@
+// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+// ------------------------------------------------------------
+
+package cosmosdb
+
+const spDefinition string = `// upserts - an array of objects to upsert
+// deletes - an array of objects to delete
+
+function sample(upserts, deletes) {
```

Comment:
can we use the meaningful name for this procedure?

Reply:
Yeah, thanks I didn't notice I didn't change it since the name isn't used when calling it.  Changed it to dapr_multi().